web/ui/module/codemirror-promql/src/grammar/promql.grammar

```diff
@@ -333,6 +335,7 @@ NumberLiteral  {
 // Contextual keywords
 
 @external extend {Identifier} extendIdentifier from "./tokens" {
+  Atan2,
```

Comment:
you need to modify the const `contextualKeywordTokens` in the file `token.js`  by adding

```javascript
atan2: Atan2
```

Reply:
Thanks! Was kinda confused why this wasn't working.

Still addressing some of the older review comments